### ISSUE TITLE
[ 1.6.0 ] Question about Units and Limits

### ISSUE THREAD

**Issue by rgipd** [viromedia/virocore#110]
Hi!

  So, I was wondering:

   1. The units in all geometry is 1 = 1 meter, just like the iOS SceneKit? In the official reference for Box, it's not showing about this (https://developer.viromedia.com/virocore/reference/com/viro/core/Box.html)

   2. The camera fieldOfView, what are the limits? In XCode SceneKit Scene Editor, we can input values from 1-179 degrees. It will adjust automatically the sensorHeight and focalLength. In the docs for Viro Camera's class, I just found that the default is 90 degrees.
  But, with the same "scene" from iOS SceneKit applied in ViroViewScene, the field of view is a lot different in values. There are any reference about this, anywhere?

  P.S.: I made a ScaleGestureDetector in Android that changes the fieldOfView camera's property to zoom in and out with pinch gesture. In iOS SceneKit I use 1 < fov < 179. In Viro ViewScene I'm using 1 < fov < 10 !!! Or the fieldOfView is using radians and not degrees?

**Comment by VikAdvani** [comment]
Hi @rgipd,
1) You are correct all units are in meters. We will amend our ViroCore docs to include this and double check we do this in other places.

2) Looking through our code I suspect you might right, we are using radians even the docs specify degrees for the fieldOfView. If you convert from degrees to radians using the following:
```
float toDegrees(float radians) {
    return radians * 180 / M_PI;
}
```
does your ScaleGestureDetector become compatible with what you see on iOS? Let me know, thanks!
I'll add this as a bug for us to take a look at.

**Comment by rgipd** [comment]
@VikAdvani the fieldOfView is indeed in degrees. I invalidated the caches and run the command gradlew clean from Android Studio and now works with the original code in degrees, just like the iOS code I had made.

Don't know if this is related with the upgrade from 1.5.0 to 1.6.0 (that I made at release date), but I think it's not, because this code I made with the 1.6.0 library.

Well, it's working as should be, but I can't explain why exactly. But the fieldOfView property is in degrees. Or in the source code from yours has two fieldOfView values somehow.


Thanks and sorry for not detected this before post the bug.
